Libraries/CustomComponents/NavigationExperimental/NavigationCard.js

```diff
@@ -87,17 +87,15 @@ class NavigationCard extends React.Component<any, Props, any> {
       ...props,
     } = this.props;
 
-    if (style === undefined) {
-      // fall back to default style.
-      style = NavigationCardStackStyleInterpolator.forHorizontal(props);
-    }
+    const defaultStyle = NavigationCardStackStyleInterpolator.forHorizontal(props);
+
     if (panHandlers === undefined) {
       // fall back to default pan handlers.
       panHandlers = NavigationCardStackPanResponder.forHorizontal(props);
     }
 
     return (
-      <Animated.View {...panHandlers} style={[styles.main, style]}>
+      <Animated.View {...panHandlers} style={[styles.main, defaultStyle, style]}>
```

Comment:
We should let people overwrite the animation style with `style`, so I don't think `defaultStyle` is necessary here.

People who use custom style should define the custom animation.

```
renderScene(props) {
  return (
    <NavigationCard 
      style={[
        NavigationCardStackStyleInterpolator.forHorizontal(props),
        styles.myStyle,
      ]}
    />
  );
}
```

This makes more sense otherwise people will start to argue that what the proper `default` style  (animation) will be and that's more like an UX question rather than an engineering question.